softperipheral/doc/sQSPI/nrf54L15_porting_v0_1_0.rst

```diff
@@ -0,0 +1,276 @@
+
+.. code-block::
+
+   nrfxlib/
+   ├──...
+   └── softperipheral/
+      ├── include
+      │   ├── softperipheral_meta.h
+      │   └── softperipheral_regif.h
+      ├── ...
+      └── sQSPI
+          ├── include
+          │   ├── hal
+          │   │   └── nrf_qspi2.h
+          │   ├── nrf54l15
+          │   │   ├── sqspi_firmware.h
+          │   │   ├── sqspi_firmware_v0.1.0.h
+          │   │   └── ...
+          │   ├── nrf_sp_qspi.h
+          │   ├── nrfx_config_qspi2.h
+          │   ├── nrfx_qspi2.h
+          └── src
+              └── nrfx_qspi2.c
+
+.. note::
+   The main interface for sQSPI is the :file:`nrfx_qspi2.h` file.
+
+Header files
+============
+
+For an sQSPI application to function correctly, it requires specific header files used by the driver code (internal paths) and specific header files for nrfx (external dependencies).
+You must include paths to these files in the build environment's list of include paths.
+The following list is a detailed breakdown of the necessary paths:
+
+* Repository internal paths:
+
+  * :file:`softperipheral/include` - Soft peripherals register interface and metadata
+  * :file:`softperipheral/sQSPI/include` - sQSPI register interface and driver header
+  * :file:`softperipheral/sQSPI/include/nrf54l15` - The sQSPI firmware for the Fast Lightweight Perpipheral Processor (FLPR)
+
+* External dependencies:
+
+   * :file:`nrfx` - Includes the main nrfx header file (:file:`nrfx.h`).
+   * :file:`nrfx/drivers/include` - Includes the nrfx driver header files.
+     For example, the :file:`nrfx_dppi.h` file.
+   * Path to the application-specific headers and macros.
+     For example, the :file:`nrfx_glue.h`, :file:`nrfx_log.h`, or a configuration header file.
+     See an example configuration for the :file:`nrfx_config.h` file:
+
+      .. code-block:: c
+
+         #ifndef NRFX_CONFIG_H__
+         #define NRFX_CONFIG_H__
+
+         #include "softperipheral_regif.h" // To Resolve correct VPR IRQn for the SoC.
+         #define nrfx_qspi2_irq_handler        SP_VPR_IRQHandler
+
+         #define NRFX_QSPI2_ENABLED            (1)
+         #define NRFX_QSPI2_MAX_NUM_DATA_LINES (4)
+
+         #endif // NRFX_CONFIG_H__
+
+Compiling source files
+======================
+
+For an sQSPI application to function properly, you must compile the driver implementation from the source file :file:`nrfx_qspi2.c`.
+
+Application core and FLPR configuration
+***************************************
+
+You must adjust the settings for the nRF54L15 SoC to run at highest base clock frequency.
+
+To work with any of the following settings, ensure you have completed the following:
+
+* You have allocated memory for the data pointers used by the sQSPI driver.
+
+  The memory is independent from the one outlined in the :ref:`nrf54L15_porting_guide_ram_configuration` subsection.
+  It is designated to function as shared memory for communication purposes, rather than containing the sQSPI executable code.
```

Comment:
```suggestion
* You have allocated memory for the data pointers used by the sQSPI driver.
  The memory is independent from the one outlined in the :ref:`nrf54L15_porting_guide_ram_configuration` subsection.
  It is designated to function as shared memory for communication purposes, rather than containing the sQSPI executable code.
```